types.js

```diff
@@ -21,7 +21,10 @@ ProjectContext.prototype.getProjectLocalDirectory = function () {
   return oldGetProjectLocalDirectory.apply(this, arguments);
 };
 
-const remoteCatalogRoot = tropohouse.root;
+let remoteCatalogRoot = tropohouse.root;
+if (process.platform == "win32") {
+  remoteCatalogRoot = remoteCatalogRoot.replace(/^\/(\w)\//, "$1:\\").replace(/\//g,'\\');  
+}
```

Comment:
Can I recommend this approach ?

```js
const remoteCatalogRoot = path.normalize(tropohouse.root);
if(process.platform === 'win32' && remoteCatalogRoot.startsWith('/')) {
  remoteCatalogRoot = remoteCatalogRoot.replace(/^\/(\w)\//, "$1:\\");
}
```

Reply:
`path.normalize()`'s behaviour tested like this :

```js
const path = require('path');

const winPath0 = String.raw`C:\Users\Me\repository\meteor\meteor-types`;
const winPath1 = String.raw`C:\Users/Me\\repository\\\meteor\\\\meteor-types`;
const winPath2 = String.raw`C:\\Users/Me\\repository\\\meteor\\\\meteor-types`;

console.log(path.normalize(winPath0)) // C:\Users\Me\repository\meteor\meteor-types
console.log(path.normalize(winPath1)) // C:\Users\Me\repository\meteor\meteor-types
console.log(path.normalize(winPath2)) // C:\Users\Me\repository\meteor\meteor-types
```

Reply:
Plugins have `Plugin.convertToOsPath` (https://docs.meteor.com/api/packagejs#build-plugin-file-system).  @ruohki if would change it to use this, I'll merge the PR.

Reply:
I didn't look too deep into this problem, just made it work for me, as I mainly work on Windows.